subsys/bluetooth/mesh/adv.h

```diff
@@ -25,40 +25,44 @@ enum bt_mesh_adv_type {
 	BT_MESH_ADV_TYPES,
 };
 
-typedef void (*bt_mesh_adv_func_t)(struct net_buf *buf, uint16_t duration,
-				   int err, void *user_data);
+enum bt_mesh_adv_tag {
+	BT_MESH_LOCAL_ADV = BIT(0),
+	BT_MESH_RELAY_ADV = BIT(1),
+
+	/* Only used with bitfield */
```

Comment:
Eliminating the secondary enumeration makes this look clearer now, thanks. However, I realised that the usage of the "bitfield" term is what caused a lot of the confusion to begin with. A bitfield (at least the way I use the term) is a variable with an explicitly stated width in bits, e.g. `uint8_t foo:2;`. And that's exactly what you're *not* using the proxy value for. I think the more appropriate term would be "bitmask" or referring to the `bt_mesh_adv.tag` struct member. However, I think you could just remove the code comment completely at this point.

Reply:
Sorry, I don’t quite understand your expression, what do I need to do?

Reply:
@LingaoM 
> remove the code comment completely at this point.